mscore/data/shortcuts.xml

```diff
@@ -687,6 +687,10 @@
     <seq>P</seq>
     </SC>
   <SC>
+    <key>toggle-lyricseditor</key>
```

Comment:
How can I disable this shortcut in Text Entry Mode to allow the user to type an "L" without displaying the Lyrics Editor? The piano shortcut "P" (just above) is disabled in text entry mode but I can't see where this happens in the code. Can someone point it out?